packages/protocol/migrations/25_federated_attestations.ts

```diff
@@ -0,0 +1,15 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
```

Comment:
Looks like this needs to be deleted in favor of migration 27 below, right?

Reply:
hey we actually need 25_federated; the 27 was part of the first version of scaffolding but is fixed in the second commit on `ASv2`
cc @isabellewei

Reply:
removing 27!